ansible-st2/roles/mistral/files/mistral.conf

```diff
@@ -1,6 +0,0 @@
-[database]
-connection=postgresql://mistral:StackStorm@localhost/mistral
```

Comment:
If we change postgresql passwords and username I guess we need to change it here as well?